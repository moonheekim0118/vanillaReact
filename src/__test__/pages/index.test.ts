import "jsdom-global/register";
import index from "../../pages/index";

describe("index page rendering", () => {
    let container;

    it("renders correctly", () => {
        container = index();
    });

    it("matches snapshot", () => {
        expect(container).toMatchSnapshot();
    });
});
