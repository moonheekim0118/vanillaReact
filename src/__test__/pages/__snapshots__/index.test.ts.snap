// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page rendering matches snapshot 1`] = `
<side
  class="container"
>
  <section
    class="Card-Container"
  >
    <div
      class="Index-Card"
    >
      <span
        class="Card-Title"
      />
      <img
        class="Image"
        src="undefined"
      />
      <span
        class="Quote"
      />
    </div>
  </section>
  <section
    class="API-Btns"
  >
    <button
      class="Generate-Btn"
    />
    <button
      class="Generate-Btn"
    />
  </section>
</side>
`;
